README.md

```diff
@@ -1,20 +1,21 @@
-# Kotatsu
+<div align="center">
+  <img src="https://raw.githubusercontent.com/KotatsuApp/Kotatsu/devel/app/src/main/res/mipmap-xxhdpi/ic_launcher_round.png" width="auto" height="auto" alt="kotatsu">
+</div>
 
 Kotatsu is a free and open source manga reader for Android.
 
 ![Android 5.0](https://img.shields.io/badge/android-5.0+-brightgreen) ![Kotlin](https://img.shields.io/github/languages/top/KotatsuApp/Kotatsu) ![License](https://img.shields.io/github/license/KotatsuApp/Kotatsu) [![weblate](https://hosted.weblate.org/widgets/kotatsu/-/strings/svg-badge.svg)](https://hosted.weblate.org/engage/kotatsu/) [![4pda](https://img.shields.io/badge/discuss-4pda-2982CC)](http://4pda.ru/forum/index.php?showtopic=697669) [![Discord](https://img.shields.io/discord/898363402467045416?color=5865f2&label=discord)](https://discord.gg/NNJ5RgVBC5)
 
-### Download
+### 📥 Download
 
-[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
-alt="Get it on F-Droid"
-height="80">](https://f-droid.org/packages/org.koitharu.kotatsu)
+<div align="center" style="width:100%; display:flex; justify-content:space-between;">
```

Comment:
Not sure if all programs that support markdown also support css and especially flex.

Reply:
It doesn't matter not gonna do any bad